docs/h2t-scc.md

```diff
@@ -0,0 +1,190 @@
+restricted        false   []   false    MustRunAs  MustRunAsRange    RunAsAny  RunAsAny
+```
+If one wishes to use any pre-defined scc, that can be done by simply adding the user or the group to the scc group.
+
+$ `oadm policy add-user-to-scc <scc_name> <user_name>`
+
+$ `oadm policy add-group-to-scc <scc_name> <group_name>`
+
+OpenShift guarantees that the capabilities required by a container are granted to the user that executes the container at admission time.  We can not allow any container to get access to unnecessary capabilities or to run in an insecure way (e.g. privileged or as root)
+
+Adding a regular user or to a group given access to the SCC allows them to run privileged pods. There are mainly four section in SCC to control access to volumes in OpenShift.
+
+1. Supplemental Groups
+1. fsGroup
+1. runAsUser
+1. seLinuxOptions
+
+### 1)Supplemental Groups
+Supplemental groups are regular Linux groups. 
+When a process runs in the system, it runs with a user ID and group ID. 
+These groups are used for controlling access to shared storage.
+
+* Check the NFS mount using the following command.
+
+   `# showmount -e <nfs-server-ip-or-hostname>`<br />
+   Export list for f21-nfs.vm: <br />
+   /opt/nfs * 
+
+* Check NFS details on the mount server using the following command.
+   
+   `# cat /etc/exports` <br />
+   /opt/nfs *(rw,sync,no_root_squash)
+
+*  `# ls -lZ /opt/nfs -d` <br />
+   drwxrws---. nfsnobody 2325 unconfined_u:object_r:usr_t:s0 /opt/nfs
+
+* `# id nfsnobody` <br />
+    uid = 65534(nfsnobody) gid = 454265(nfsnobody) groups = 454265(nfsnobody)
+    The /opt/nfs/ export is accessible by UID 454265 and the group 2325.
+
+    ```
+    apiVersion: v1
+    kind: Pod
+    ...
+    spec:
+       containers:
+       - name: ...
+          volumeMounts:
+          - name: nfs
+             mountPath: /usr/share/...
+       securityContext:
+          supplementalGroups: [2325]
+       volumes:
+       - name: nfs
+          nfs:
+          server: <nfs_server_ip_or_host>
+          path: /opt/nfs
+    ```
+
+### 2)fsGroup
+
+fsGroup stands for the file system group which is used for adding container supplemental groups. 
+Supplement group ID is used for shared storage and fsGroup is used for block storage.
+
+```
+kind: Pod
+spec:
+   containers:
+   - name: ...
+   securityContext:
+      fsGroup: 2325
+```
+
+### 3)runAsUser
+
+runAsUser uses the user ID for communication. 
+This is used in defining the container image in pod definition. A single ID user can be used in all containers, if required.
+While running the container, the defined ID is matched with the owner ID on the export. 
+If the specified ID is defined outside, then it becomes global to all the containers in the pod. 
+If it is defined with a specific pod, then it becomes specific to a single container.
+
+```
+spec:
+   containers:
+   - name: ...
+      securityContext:
+         runAsUser: 454265
+```
+
+### 4)seLinuxOptions
```

Comment:
```suggestion
### 4. seLinuxOptions
```

Reply:
I modified the line to` ### 4. seLinuxOptions` but still it is showing as `### 4)seLinuxOptions`.  Previous  headers  showing in right sequence.

Reply:
oh, It is showing now correctly.